nexus/auth/src/authn/external/mod.rs

```diff
@@ -9,6 +9,7 @@ use super::SiloAuthnPolicy;
 use crate::authn;
 use async_trait::async_trait;
 use authn::Reason;
+use slog::trace;
```

Comment:
i'm assuming this is because there used to be a `#[macro_use] somewhere?

Reply:
Exactly: https://github.com/oxidecomputer/omicron/blob/b0dfd535386cb93361e08a17d926e91867d9cb08/nexus/db-queries/src/lib.rs#L14-L17